content/sources/contributions.adoc

```diff
@@ -11,23 +11,20 @@ Creating a PR is easy and allows to share, discuss and validate more easily the
 To create a PR :
 
 * create a branch locally with your fix and push it to Github
-* in Github, select your branch, and click on *New pull request* image:git-pr-create-01.png[alt="Git PR Create",align="center",width="700"]
 
-* select the right base branch (most of the time *develop*) and check that the PR contains only your commits image:git-pr-create-02.png[alt="Git PR Create",width="700",align="center"]
-* fill the description ! See <<id-pull-requests-best-practices>> for more details.
-* click on *Create pull request*
+* go to Github, and select your branch, to create New pull request
+
+* select the right base branch (`develop`, `feature/`, `stable/`...) and check that the PR contains *only your commits*
 
-PR must always be done on `exodev:develop` or `exo-addons:develop`, not on `exoplatform:develop` (we fix on develop first, then we backport to stable if needed).
-There are only 2 exceptions :
+* fill the description ! See Pull Requests best practices for more details.
```

Comment:
Should we put a link on "Pull Requests best practices" ?

Reply:
Right! moved to OnlyOffice Documents

Reply:
The reference has already been mentioned in the previous paragraph, so this sentence will be simply removed to avoid repetition.